src/api/authService.ts

```diff
@@ -0,0 +1,66 @@
+import Cookies from "js-cookie";
+import { COOKIE_EXPIRY } from "@/constants/auth";
+
+const API_BASE_URL =
+  import.meta.env.VITE_API_BASE_URL || "http://localhost:3000";
+
+export const authService = {
+  // Redirect to Google OAuth (backend ready)
+  initiateGoogleLogin: (): void => {
+    window.location.href = `${API_BASE_URL}/api/auth/google`;
+  },
+
+  // Redirect to LinkedIn OAuth (backend in progress)
+  initiateLinkedInLogin: (): void => {
+    window.location.href = `${API_BASE_URL}/api/auth/linkedin`;
+  },
```

Comment:
delete comments, they are not needed. 
does it posible to make it without window.location.href? I dont like windows manipulation and I know that its technically posible